ocean_lib/assets/test/test_asset.py

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from ocean_lib.config_provider import ConfigProvider
+from tests.resources.ddo_helpers import (
+    wait_for_ddo,
+    get_registered_ddo_with_compute_service,
+    get_registered_algorithm_ddo,
+)
+from tests.resources.helper_functions import get_publisher_wallet
+
+
+def test_values(publisher_ocean_instance, metadata):
+    """Test the value property."""
+    publisher = get_publisher_wallet()
+    metadata_copy = metadata.copy()
+
+    ddo = publisher_ocean_instance.assets.create(metadata_copy, publisher)
+    wait_for_ddo(publisher_ocean_instance, ddo.did)
+
+    ddo_values = ddo.values
+    assert ddo_values is not None
+    for key, value in ddo_values.items():
+        assert key == "dataToken"
+        assert ddo_values[key] is not None
```

Comment:
you can also test the format, e.g. length, starts with 0x (if that is the case). It would be a bit more thorough.